src/components/app/ListOfPublishedProfiles.js

```diff
@@ -344,6 +344,20 @@ export class ListOfPublishedProfiles extends PureComponent<Props, State> {
     const profilesRestCount =
       profileDataList.length - reducedProfileDataList.length;
 
+    const profileListLabel =
```

Comment:
I think we can improve this code by not initializing this `profileListLabel` variable if profilesRestCount is not more than zero. Right now we are initializing a Node even though we are not going to use it. My recommendation would be doing something like this:
```js
let profilesRestLabel;
if (profilesRestCount > 0) {
  profilesRestLabel =  (<>See and manage all your recordings ({profilesRestCount} more)</>);
} else {
  profilesRestLabel =
      profileDataList.length > 1 ? (
        <>Manage these recordings</>
      ) : (
        <>Manage this recording</>
      );
}
```
What do you think? Would you mind changing that before landing the PR?

Reply:
Sure changing it.